lib/9T/magic/ext4mag.py

```diff
@@ -0,0 +1,195 @@
+#!/usr/bin/env python
```

Comment:
is that different from the `12T` version?

Reply:
not needed - deleted.